organizations (#782)
neo4j/docs-aura: modules/ROOT/pages/visual-tour/index.adoc

== Organizations

Organizations are a layer in the Aura account hierarchy that sits above projects.
This allows you to organize and manage multiple projects under a single organization.

> Management at this level helps `Organization owners` and `Organization admins` roll out settings to all projects and instances within an organization.

Reviewer: ```suggestion
Management at this level allows `Organization owners` and `Organization admins` roll out settings to all projects and instances within an organization.
```

Reviewer: Although maybe we shouldn't talk about multiple projects since that is not currently possible?

Reviewer: Here, I'd suggest "Management at this level allows `Organization owners` and `Organization admins` roll out settings to projects and instances within an organization."

Just to de-emphasize how many projexts you can have.

To access organization-level management, click the organization name at the top of the page. 

.Organization navigation
[.shadow]
image::organizationnav.png[]

> From there, use the left-hand panel to navigate to key sections:

Reviewer: ```suggestion
From there, use the left-hand panel to navigate:
```

> * xref:visual-tour/index.adoc#_projects[Projects]: View and manage all projects within the organization.

Reviewer: Same here, it's not currently possible to have more than one project.

Author: @AlexicaWright - what do you suggest?

Reviewer: Same here, maybe just remove *all* is enough? This is bad design and we can't fix it with good docs 😆 , but I assume it's temporary and that multiple projects will be reality some day soon.

> * Users: View user emails and their xref:user-management.adoc#_organization_level_roles[organizational-level roles]

Reviewer: ```suggestion
* Users: View users and their xref:user-management.adoc#_organization_level_roles[organization-level roles]
```

* Security: Configure organization-level features such as SSO, IP filtering, and tool authentication with Aura user.

> * xref:visual-tour/index.adoc#org-settings[Settings]: Also called *Organization Settings*, this is where you can manage additional settings. 

Reviewer: ```suggestion
* xref:visual-tour/index.adoc#org-settings[Settings]: Where you can manage general organization settings, preferences and feature access. 
```

[[org-settings]]
=== Organization settings

> You can edit the name of the organization, find your organization ID, and enable Generative AI and Aura Graph Analytics across your teams.

Reviewer: Which teams?
